### PR TITLE
Feature/admin login button

### PR DESCRIPTION
**Pull request by aliroemhildt**
## Checklist
- [ ] Tests written where required
- [ ] All tests passing
- [ ] Unused code/prys/etc deleted
Testing Coverage at <percentage>%

## Changes Made
  
What (if any) features are you implementing?
- login admin 1 on login button click
- conditionally render select winner button if admin is logged in
- update winner on select winner button click (based on what the query is returning this seems to be working)
- fetch winner and display on landing page (this is only returning customer1, not sure if this query is working)

What (if anything) did you refactor?
- n/a

Were there any issues that arose?
- not sure if the getWinner query is working, it seems like it might just be returning a hardcoded winner

Is there anything that you need from your teammate?
- n/a

Any other comments, questions, or concerns?
- n/a

### CODE REVIEW

File: src/components/LandingPage/LandingPage.js [hunk 1]
```diff
@@ -1,10 +1,28 @@
...
+        console.log(response.data.getWinner)
+        setWinner(response.data.getWinner)
+      })
+  }, [])
+
+  const { loading, error, data, refetch } = useQuery(GET_WINNER)
+  
+  const displayWinner = winner &&
+    <p className="winner">{data.getWinner.name} is the winner!!!!!!!</p>
+
```
Comment (mr-ryan12) [inline src/components/LandingPage/LandingPage.js:22]: Thank you for implementing this! We can meet with BE folks and find out if the mutation is updating the admin.

File: src/components/LandingPage/LandingPage.scss [hunk 2]
```diff
@@ -6,20 +8,23 @@
...
+  width: 80%;
   text-align: center;
-  position: absolute;
-  top: 30%;
-  left: 50%;
-  transform: translate(-50%, -50%);
+  // position: absolute;
+  // top: 30%;
+  // left: 50%;
+  // transform: translate(-50%, -50%);
```
Comment (mr-ryan12) [inline src/components/LandingPage/LandingPage.scss:27]: Do we need this?

File: src/components/NavBar/NavBar.js [hunk 3]
```diff
@@ -1,17 +1,32 @@
...
+  const handleLogin = () => {
+    refetch();
+    if (!user && data) {
+      setUser(data.getAdmin);
+    } else if (user) {
+      setUser(null);
+    }
+  // need to error handling for error/loading
+  }
+
```
Comment (mr-ryan12) [inline src/components/NavBar/NavBar.js:23]: Totally cool we figured out how this refetch works!

**Review by mr-ryan12 — APPROVED** [review]

**Review by oacortez — APPROVED** [review]
Thank you for creating this branch, Ali! I see we got some mutation going! I see we are able to  log in an admin, and they are able to select a winner. I see we are not able to display the winner once we are refetching that's something we need to discuss with our BE team other than that everything looks good. Thank you, Ali!

**Pull request merged**